src/api/mod.rs

```diff
@@ -680,7 +712,13 @@ impl<T: Pixel> Context<T> {
     ))
   }
 
-  /// Send the information to the encoder
+  /// Sends the frame for encoding.
+  ///
+  /// It could return [`EncoderStatus::EnoughData`] if the incoming frame queue
+  /// is full or [`EncoderStatus::Failure`] if the per-frame parameters set an
+  /// impossible constraint.
```

Comment:
This is not accurate right now, since `EnoughData` is literally never returned, despite existing as a possible status.

Reply:
What do you mean? It returns `EnoughData` right here:
https://github.com/xiph/rav1e/blob/dc34123f5316c66547d3cfbfb86855991db1a1ec/src/api/mod.rs#L722

Reply:
Ah, it certainly never used to until a few weeks back.

However... now it seems that `EnoughData` is used during flushing, and still not for when the incoming frame queue is full (which to my knowledge, cannot happen at the moment).